### ISSUE TITLE
phapp loads .env files in incorrect order.

### ISSUE THREAD

**Issue by rthideaway** [drunomics/phapp-cli#18]
There may be a case when you can have several env files and some of them rely on variables in different files. So the order of loading is important. Example:

```
.example-1.env:
EXAMPLE_SUFFIX=some-suffix
```

```
.example-2.env:
EXAMPLE_DOMAIN=example-${SUFFIX}.com
```
If the second file is loaded first then you end up with incorrect value, where EXAMPLE_DOMAIN being just `example-.com` instead of `example-some-suffix.com`. That's what is hapenning with phapp env commands. Would be great if could agree on some specific rules how to load the .env files so we can make sure they are loaded as it's inteded.

**Comment by rthideaway** [comment]
I rolled [PR](https://github.com/drunomics/phapp-cli/pull/19) where the env files will be sorted alphabetically before loaded so there is at least some sort of loading order. Example of env files loading order:
```
(before PR):
.d.env
.a.env
.exlude.env
.site.env
.dd.env
.common.env
.b.env
.random.env
.env
.xss.env
.ssl.env
.shared.env
.test.env
.example.env
.c.env
.another.env
.ddd.env
.ccc.env
.cookie.env
.unique.env
```
```
(after PR):

.a.env
.another.env
.b.env
.c.env
.ccc.env
.common.env
.cookie.env
.d.env
.dd.env
.ddd.env
.env
.example.env
.exlude.env
.random.env
.shared.env
.site.env
.ssl.env
.test.env
.unique.env
.xss.env
```

Additionally I had to populate variables after particular env file is loaded so the variable are presented for next env file as well.

**Comment by rthideaway** [comment]
One more commit to parse the vars without populating them. Example of files which were tested.
```
.a.env:
A=a
```
```
.b.env:
B=${A}bb
```
```
.c.env:
C=${A}${B}ccc
```
```
D=${A}${B}${C}ddd
```
Results of parse variables:
```
array(4) {
  'A' =>
  string(1) "a"
  'B' =>
  string(3) "abb"
  'C' =>
  string(7) "aabbccc"
  'D' =>
  string(14) "aabbaabbcccddd"
}
```

**Comment by fago** [comment]
This is resolved by the recent 0.6.0 release - it now does not load it any more but leave it up to you to load it as you want :)
